docs/install/harvester-configuration.md

```diff
@@ -63,6 +63,7 @@ install:
   iso_url: http://myserver/test.iso
   poweroff: true
   no_format: true
+  datadisk: /dev/sdb
```

Comment:
IIRC the `data_disk` should be the correct name. cc @bk201 to double confirm this.
https://github.com/harvester/harvester-installer/blob/v1.1/pkg/config/config.go#L120

Reply:
yeah, it should be `data_disk`. @clemenko, does `data_disk` work for you?

Reply:
It didn't a month ago. I need to re-figure my test scenario.

Reply:
> IIRC the `data_disk` should be the correct name. cc @bk201 to double confirm this. https://github.com/harvester/harvester-installer/blob/v1.1/pkg/config/config.go#L120

no where in this file does it use `data_disk`?